packages/email-templates/src/templates/keyMined.ts

```diff
@@ -1,8 +1,10 @@
 import handlebars from 'handlebars'
 import { customContentStyle } from './helpers/customContentStyle'
 import { links } from './helpers/links'
+import { eventDetails } from './helpers/eventDetails'
```

Comment:
ok so rather than re-using the `keyMined` template here we should have a whole new template `eventKeyMined` that locksmith calls ... (with failover as `keyMined`)